cmd/eval-dev-quality/cmd/evaluate_test.go

```diff
@@ -91,6 +92,11 @@ func validateMetrics(t *testing.T, regex *regexp.Regexp, data string, expectedAs
 }
 
 func TestEvaluateExecute(t *testing.T) {
+	if osutil.IsLinux() {
```

Comment:
@ahumenberger it would be better to have a testing solution that is in one place instead of multiple. If you have ideas. At the moment, i don't

Reply:
yeah... I tried to come with one but also... kinda out of ideas...

Reply:
Only way I could think of would be to introduce a `testing.T`  wrapper that we need to use everywhere instead of the normal one. And that one could then probably to the tools check automatically every time.

Reply:
> Only way I could think of would be to introduce a `testing.T` wrapper that we need to use everywhere instead of the normal one. And that one could then probably to the tools check automatically every time.

Too complicated.

Another solution is to have a testing package and do it there in an "init" function. Would lead to having all these checks only once.

Reply:
where would this testing package be? or do you mean moving all the `_test.go` files into a separate package? I hope that we never access any private things cause otherwise that would not be doable.

Reply:
Let's rework this another time.